### PR TITLE
fixed main.bicep

### PR DESCRIPTION
**Pull request by hunter-walden2113**
## Purpose
Updating the main.bicep to fit our deployment needs. 
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X ] Other... Please describe: updating pre-existing bicep template
```

## How to Test
Not Required

## What to Check
Verify that the following are valid:
- parameter settings
- module settings

## Other Information
N/A

### CODE REVIEW

File: infra/main.bicep [hunk 1]
```diff
@@ -115,10 +71,9 @@ module backend 'core/host/appservice.bicep' = {
 
 module openAi 'core/ai/cognitiveservices.bicep' = {
   name: 'openai'
-  scope: openAiResourceGroup
   params: {
     name: !empty(openAiServiceName) ? openAiServiceName : '${abbrs.cognitiveServicesAccounts}${resourceToken}'
```
Comment (jhaugh42) [inline infra/main.bicep:75]:
```suggestion
    name: openAiServiceName
```

File: infra/main.bicep [hunk 2]
```diff
@@ -49,33 +30,9 @@ var abbrs = loadJsonContent('abbreviations.json')
 var resourceToken = toLower(uniqueString(subscription().id, environmentName, location))
 var tags = { 'azd-env-name': environmentName }
```
Comment (jhaugh42) [inline infra/main.bicep:31]: After tags are gone and the ternary name assignments are gone, these two vars can be eliminated.

File: infra/main.bicep [hunk 3]
```diff
@@ -115,10 +71,9 @@ module backend 'core/host/appservice.bicep' = {
 
 module openAi 'core/ai/cognitiveservices.bicep' = {
   name: 'openai'
-  scope: openAiResourceGroup
   params: {
     name: !empty(openAiServiceName) ? openAiServiceName : '${abbrs.cognitiveServicesAccounts}${resourceToken}'
-    location: openAiResourceGroupLocation
+    location: location
     tags: tags
```
[the remark this answers is not in the document]
Comment (hunter-walden2113) [reply infra/main.bicep:77]: complete. all tags removed from main.bicep.

File: infra/main.bicep (hunk 1 above)
Comment (hunter-walden2113) [reply infra/main.bicep:75]: complete. removed the logic to check if name is empty through all of main.bicep.

**Review by jhaugh42 — APPROVED** [review]

**Pull request merged**